lib/blocks/src/blocks/Docs.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import type { FunctionComponent, ComponentType } from 'react';
+import type { AnyFramework, Parameters } from '@storybook/csf';
+import type { Theme } from '@storybook/theming';
+
+import type { DocsContextProps } from './DocsContext';
+import { DocsContainer } from './DocsContainer';
+import { DocsPage } from './DocsPage';
+
+export type DocsProps<TFramework extends AnyFramework = AnyFramework> = {
+  parameters: Parameters;
```

Comment:
```suggestion
  docsParameter: Parameters;
```

Reply:
I found this confusing when reading the code. Usually I expect `parameters` to refer to the full set of parameters, rather than just the scoped docsParameter. Hence my suggestion.